doc/client_lua_api.md

```diff
@@ -131,7 +131,7 @@ The `:` prefix can also be used for maintaining backwards compatibility.
 
 Sounds
 ------
-**NOTE: Not fully implemented yet.**
+**NOTE: max_hear_distance and connecting to objects is not implemented.**
```

Comment:
you didn't comment out the `object` section below, does this mean that the sound will play in the *current* position of the object, and won't move with the object?